gapic/src/main/com/google/gapid/perfetto/views/StyleConstants.java

```diff
@@ -43,6 +43,7 @@
   public static final double SELECTION_THRESHOLD = 0.333;
   public static final double ZOOM_FACTOR_SCALE = 0.05;
   public static final double ZOOM_FACTOR_SCALE_DRAG = 0.01;
+  public static final double FLAGS_Y = 24;
```

Comment:
Nit: I would suggest put this one in RootPanel.java along with those highlight constants, since this one is not shared cross different files, or extremely global, like the other constants here.